generator/app.go

```diff
@@ -244,7 +269,7 @@ func createGroupReadme(groups []Group, prefix string) error {
 
 		outputPath := filepath.Join(outputDir, indexFilename)
 		readmePath := filepath.Join(baseGeneratorDir, templateDir, fmt.Sprintf("%s_%s", prefix, readmeTemplate))
-		if err := writeTemplate(readmePath, outputPath, true, group); err != nil {
+		if err := writeTemplate(readmePath, outputPath, "markdown", group); err != nil {
```

Comment:
instead of adding a new arg, could we compute this by reading the path extension?

Reply:
`OWNERS_ALIASES` doesn’t have an extension :(